### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by jaakkonakaza**
## Description

Noticed the heading below says Changsets instead of Changesets so used a proofreading plugin to find some typos

## Changsets

We use [Changesets](https://github.com/changesets/changesets) to automatically create our changelog per each release. Any changes or additions to the Library assets in `/lib` must be documented with a new Changeset. This can be done as follows:

2. Navigate to the root of the project on your feature branch.
3. Run `pnpm changeset` to trigger the Changeset CLI.
4. Follow the instructions when prompted.
    - Changesets should be either `minor` or `patch`. Never `major`.
    - Prefix your Changeset description using: `feature:`, `chore:` or `bugfix:`.
5. Changeset `.md` files are added to the `/.changeset` directory.
6. Commit and push the the new changeset file.

## Checklist

Please read and apply all [contribution requirements](https://github.com/skeletonlabs/floating-ui-svelte/blob/chore/main/CONTRIBUTING.md).

- [x] PR targets the `dev` branch (NEVER `master`)
- [x] All website documentation is current with your changes
- [x] Ensure Prettier formatting is current - run `pnpm format`
- [x] Ensure Prettier linting is current - run `pnpm format`
- [x] All test cases are passing - run `pnpm test`
- [ ] Includes a changeset (if relevant; see above)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,5 +68,6 @@
 	},
 	"svelte": "./dist/index.js",
 	"types": "./dist/index.d.ts",
-	"type": "module"
+	"type": "module",
+	"packageManager": "pnpm@9.1.0+sha512.67f5879916a9293e5cf059c23853d571beaf4f753c707f40cb22bed5fb1578c6aad3b6c4107ccb3ba0b35be003eb621a16471ac836c87beb53f9d54bb4612724"
```
Comment (endigo9740) [inline package.json:72]: I believe this is something you keep fighting to remove. Might be worth looking into a better long term solution.

**Review by endigo9740 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jaakkonakaza) [reply package.json]: I moved it into another changelist in WebStorm now so that should keep me from committing it again

Comment (jaakkonakaza) [reply package.json]:
```
pnpm dev
! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing pnpm@9.1.0+sha512.67f5879916a9293e5cf059c23853d571beaf4f753c707f40cb22bed5fb1578c6aad3b6c4107ccb3ba0b35be003eb621a16471ac836c87beb53f9d54bb4612724.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager
```

The hash is added by Corepack

**Review by Hugos68 — APPROVED** [review]
lgtm, thanks!
PS: I see the checklist is left untouched, you can actually check these, this also notifies us you've covered them.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -25,6 +25,6 @@ Please read and apply all [contribution requirements](https://github.com/skeleto
 - [ ] PR targets the `dev` branch (NEVER `master`)
 - [ ] All website documentation is current with your changes
 - [ ] Ensure Prettier formatting is current - run `pnpm format`
-- [ ] Ensure Prettier linting is current - run `pnpm format`
+- [ ] Ensure Prettier linting is current - run `pnpm lint`
```
Comment (Hugos68) [inline .github/pull_request_template.md:28]: This should be `Ensure ESLint linting is current` if we're going to mention the tool.

**Review by Hugos68 — APPROVED** [review]

**Review by endigo9740 — APPROVED** [review]

**Comment by endigo9740** [comment]
@Hugos68 @jaakkonakaza I've given my approval, but looks like a minor merge conflict will need to be resolved here.

**Comment by Hugos68** [comment]
I've resolved the conflict, I'll merge once CI passes👍

**Pull request merged**